NewKernel_d/include/CGAL/NewKernel_d/Cartesian_filter_K.h

```diff
@@ -63,6 +63,12 @@ struct Cartesian_filter_K : public Base_
     typedef typename Store_kernel<EK_>::reference_type EK_rt;
     EK_rt exact_kernel()const{return sek.kernel();}
 
+
+    // Added to be similar to CGAL::Filtered_kernel
+    typedef EK Exact_kernel;
+    enum { Has_filtered_predicates = true };
+    typedef Boolean_tag<Has_filtered_predicates> Has_filtered_predicates_tag;
+
```

Comment:
@mglisse please review

Reply:
As I said elsewhere, if that's really all that's needed (there is an example so I assume it compiles and runs, although I am rather surprised), ok.
(I would have mentioned in the comment the name of the package that needs it, but I guess this comment is good enough)